pkg/kubectl/resource_printer.go

```diff
@@ -631,6 +634,21 @@ func printResourceQuotaList(list *api.ResourceQuotaList, w io.Writer) error {
 	return nil
 }
 
+func printServerStatus(item *api.ServerStatus, w io.Writer) error {
+	_, err := fmt.Fprintf(w, "%s\t%s\t%d\t%s\t%s\n", item.Component, item.Health, item.HealthCode, item.Msg, item.Err)
+	return err
+}
+
+func printServerStatusList(list *api.ServerStatusList, w io.Writer) error {
+	for _, item := range list.Items {
+		if err := printServerStatus(&item, w); err != nil {
```

Comment:
It might be better not to short-circuit here but to print all of the status lines and then return an error at the end if any individual line caused an error. That way if there is more than one error you can see all of them at once.